tests/app/v2/test_errors.py

```diff
@@ -128,7 +128,5 @@ def test_bad_method(app_for_test):
 
             assert response.status_code == 405
 
-            assert response.json == {
-                "result": "error",
-                "message": "The method is not allowed for the requested URL."
-            }
+            assert response.get_data(as_text=True) == \
```

Comment:
what happened with this assertion? Sorry if I'm asking silly questions, it's just confusing 😅

Reply:
The test request response no longer has json as an attribute.
All the changes in this PR are because the dependencies were incremented.